### ISSUE TITLE
Use aria-current to indicate the currently selected tab

### ISSUE THREAD

**Issue by joedolson** [WordPress/health-check#286]
### Feature request/bug description

The attribute 'aria-current' indicates to a screen-reader that a link is the currently selected of its set of links. See https://tink.uk/using-the-aria-current-attribute/

In this case, I would use `aria-current="true"`, as the currently selected page should be 'Site Health'.
